src/main.js

```diff
@@ -147,9 +150,9 @@ export default class Auth {
 	}
 
 	/**
-	 * Updates the user data in localStorage.
-	 * @param {Object} userData New user data.
-	 * @param {boolean} [updateStorage = true] Check whether to update localStorage or not.
+	 * Updates the user data in the localStorage.
+	 * @param {Object} userData The new user data.
+	 * @param {boolean} [updateStorage = true] Whether to update local storage or not.
```

Comment:
Check is essential here, otherwise the sentence is incomplete.

Also, I did not think about storage method can be replaced and may not be localStorage exclusively.

Maybe, "localStorage" should be replaced by "browser storage" in the comments?

Reply:
This lib is used by some outside of the browser, that's why I try to avoid using any other term.
And `localStorage` is the name of the storage we are currently using by default, so I think its  OK to use.

As for "check". I think the change is OK.